src/runAll.js

```diff
@@ -24,44 +25,35 @@ module.exports = function runAll(scripts, config) {
   const renderer = config.renderer
   sgf.cwd = resolveGitDir(gitDir)
 
-  return new Promise((resolve, reject) => {
-    sgf('ACM', (err, files) => {
-      if (err) {
-        return reject(err)
```

Comment:
It seems this part missing in the new version or how is this error going to be handled?

Reply:
Since we are promisifying `sgf`, the `Promise` returned will reject with the error if there was one.

Reply:
We also have a test for this which is passing:
https://github.com/okonet/lint-staged/blob/1f0b376cfa74d2565131ca1a0032cd1c8ebb53a4/test/runAll.spec.js#L56-L62